### PR TITLE
Update reference overview page

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: reference/index.md [hunk 1]
```diff
@@ -5,38 +5,148 @@ notoc: true
...
+<div class="component-container">
+<!--start row-->
+    <div class="row">
+     <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+            <div class="component-icon">
+                 <a href="/engine/reference/commandline/cli/"><img src="/assets/images/terminal.svg" alt="Docker CLI" width="70" height="70"></a>
+            </div>
+                <h2 id="docker-for-linux"><a href="/engine/reference/commandline/cli/">Docker CLI</a></h2>
+                <p>The main CLI for Docker, includes all `docker` commands.</p>
```
Comment (craig-osterhout) [inline reference/index.md:47]: `docker` isn't formatted.

**Review by craig-osterhout — COMMENTED** [review]
It looks good to me. If everyone is okay with this style, I can update the Guides overview to follow the same style with titles for the top-level sections and cards for the sub-topics.

File: reference/index.md (hunk 1 above)
Comment (dvdksn) [reply reference/index.md:47]: wrapping "docker" in a `<code>` element should work

**Comment by dvdksn** [comment]
Really nice change 👍🏻

File: reference/index.md (hunk 1 above)
Comment (aevesdocker) [inline reference/index.md:47]:
```suggestion
                <p>The main CLI for Docker, includes all <code>docker</code> commands.</p>
```

Comment (aevesdocker) [reply reference/index.md]: Woops, thanks for catching that!

**Comment by aevesdocker** [comment]
@craig-osterhout yeh I think it would be a good structure to apply to the guides section. At the same time though, since we now have a date for the Hugo move, maybe you don't want to waste the energy?

**Review by craig-osterhout — APPROVED** [review]

**Pull request merged**